organisations/README.md

```diff
@@ -0,0 +1,122 @@
+| identifier\[0].system                             | fixed value = `"https://fhir.nhs.uk/Id/ods-organization-code"` |
+| identifier\[0].value                              | The ODS code of the losing practice                            |
+
+### Unmapped fields
+
+The following Organisation fields are not currently populated by the adaptor: 
+
+- implicitRules
+- language
+- contained
+- extension
+- identifier.id
+- identifier.use
+- identifier.type
+- identifier.type.id
+- identifier.period
+- identifier.assigner
+- active
+- type.id
+- type.coding
+- alias
+- telecom.id
+- telecom.period
+- address.id
+- address.text
+- address.city
+- address.district
+- address.country
+- address.period
+- partOf
+- contact
+- endpoint
+
+## JSON FHIR > XML HL7
+
+GP Connect FHIR Organisations are mapped to the HL7v3 Agent Directory.  
+
+Where an Organisation is not a mapped as a `representedOrganisation` of an `AgentPerson` ([Practitioner](../practioners/README.md)), the adaptor maps the 
+Organisation to an `AgentPerson` instead of `AgentOrganisation`. This is due to compatability issues where attribution 
+was given to the Organisation as a performer.
+
+### Organisation as an AgentPerson 
+| Mapped to (XML HL7)         | Mapped from (JSON FHIR / other source) |
+|-----------------------------|----------------------------------------|
+| Agent / id \[@root]         | New UUID gnerated by the adaptor       |
+| Agent / code \[@nullFlavor] | fixed value = `"UNK"`                  |
+| Agent / code / originalText | fixed value = `Unknown`                |
+| Agent / name / family       | `Organisation.name`                    |
+
+### Represented Organisation
+| Mapped to (XML HL7)                                        | Mapped from (JSON FHIR / other source)                                   |
+|------------------------------------------------------------|--------------------------------------------------------------------------|
+| Agent / representedOrganisation / name                     | `Organisation.name`                                                      | 
+| Agent / representedOrganisation / telecom                  | `Organisation.telecom.value` <sup>4</sup>                                |
+| Agent / representedOrganisation / addr / streetAddressLine | `Organisation.address.line` and `Organisation.address.city` <sup>5</sup> |
+| Agent / representedOrganisation / addr / postalCode        | `Organisation.address.postalCode` <sup>5</sup>                           |  
+
+
+4. The first instance of `Organisation.telecom` where `Organisation.telecom.system.display` is equal to `"phone"` and `Organisation.telecom.use` is equal to `"work"`   
+5. Where `Organisation.address` is the first instance. 
+
+## Further documentation
+[GP Connect Organisation Structure definition](https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Organization-1)
```

Comment:
Is it me, or does this website not render properly.

<img width="146" alt="Ugly looking website" src="https://github.com/NHSDigital/patient-switching-adaptors-mapping-documentation/assets/976274/fb6bde58-625b-42cc-b31c-9597abbb4d56">